src/commands/createFunctionApp/FunctionAppCreateStep.ts

```diff
@@ -80,6 +80,16 @@ export class FunctionAppCreateStep extends AzureWizardExecuteStep<IFunctionAppWi
             this.addCustomLocationProperties(site, context.customLocation);
         }
 
+        // Always on setting added for App Service plans excluding the free tier https://github.com/microsoft/vscode-azurefunctions/issues/3037
+        if (context.plan?.sku?.family !== undefined) {
```

Comment:
I believe we can remove the `!== undefined`

```suggestion
        if (context.plan?.sku?.family) {
```